apps/mudita-center-e2e/src/specs/news/news-check-offline.e2e.ts

```diff
@@ -0,0 +1,131 @@
+import HomePage from "../../page-objects/home.page"
+import NewsPage from "../../page-objects/news.page"
+import {
+  commentsRegex,
+  linkRegex,
+  newsDateRegex,
+  newsImageRegex,
+} from "../../consts/regex-const"
+
+describe("News Page Check in Offline Mode", () => {
+  before(async () => {
+    // Clear browser cache
+    await browser.deleteAllCookies()
+    await browser.execute("window.localStorage.clear();")
+    await browser.execute("window.sessionStorage.clear();")
+
+    // Switch to offline mode before starting the tests
+    await browser.setNetworkConditions({
+      offline: true,
+      latency: 0,
+      download_throughput: 0,
+      upload_throughput: 0,
+    })
+
+    // Add a small delay to ensure network conditions are applied
+    await browser.pause(1000)
+
+    // Verify network conditions
+    const isOnline = await browser.execute(() => navigator.onLine)
```

Comment:
I am thinking about another thing - moving this to `helpers/tests.helper.ts` because you use it two times

Reply:
added isOnline() {
    return browser.execute(() => navigator.onLine)
  } to testsHelper 
  
  And  updated verification of network conditons as:
  // Verify network conditions
    const isOnline = await testsHelper.isOnline()
    await expect(isOnline).toBeFalsy()
    
    // Verify network conditions
    const isOnline = await testsHelper.isOnline()
    await expect(isOnline).toBeTruthy()